enterprise-suite/console_test/console_test.go

```diff
@@ -0,0 +1,98 @@
+package console
+
+import (
+	"fmt"
+	"testing"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/lightbend/console_test/args"
+
+	"github.com/lightbend/console_test/util/helm"
+	"github.com/lightbend/console_test/util/lbc"
+	"github.com/lightbend/console_test/util/minikube"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+// The following variables are used in this package to access kubernetes
+// and make requests on Console components.
+var k8sClient *kubernetes.Clientset
+var consoleAddr string
+var prometheusAddr string
+var monitorAPIAddr string
+
+var _ = BeforeSuite(func() {
+	// Start minikube if --start-minikube arg was given
+	if args.StartMinikube {
+		Expect(minikube.IsRunning()).ShouldNot(BeTrue())
+		Expect(minikube.Start(3, 6000)).To(Succeed())
+	}
+
+	// Setup k8s client
+	config, err := clientcmd.BuildConfigFromFlags("", args.Kubeconfig)
+	if err != nil {
+		panic(err.Error())
+	}
+	k8sClient, err = kubernetes.NewForConfig(config)
+	if err != nil {
+		panic(err.Error())
+	}
+
+	// If we started minikube, install helm as well
+	if args.StartMinikube {
+		Expect(helm.Install(k8sClient)).To(Succeed())
```

Comment:
Maybe installing helm should be a separate flag? Although I'm not sure we should be doing this in the tests. I wonder if we should just rely on helm being setup already. :conflicted:

Reply:
Kept this all under `--start-minikube` for now, but I'm open to changes. I find some value in being able to fire and forget a test suite without any manual env setup, so I like that `console_test` can start minikube & helm by itself. Other workflows may wary of course!